servers/rendering/renderer_rd/shaders/scene_forward_lights_inc.glsl

```diff
@@ -275,7 +275,7 @@ float sample_directional_pcf_shadow(texture2D shadow, vec2 shadow_pixel_size, ve
 
 	mat2 disk_rotation;
 	{
-		float r = quick_hash(gl_FragCoord.xy) * 2.0 * M_PI;
+		float r = quick_hash(gl_FragCoord.xy + vec2(taa_frame_count * 5.588238)) * 2.0 * M_PI;
```

Comment:
Was there a specific reason we went for `5.588238`? I can't find this value anywhere else in the codebase. For context, I'm asking because of https://github.com/godotengine/godot/pull/101961.

Reply:
It's the value suggested by the author of IGN. There is a nice explanation here: https://blog.demofox.org/2022/01/01/interleaved-gradient-noise-a-different-kind-of-low-discrepancy-sequence/